### PR TITLE
Add Cellana Finance

### PR DESCRIPTION
**Pull request by Centure2222**

### CODE REVIEW

File: data/ecosystems/c/data/ecosystems/m/cellana-finance.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Ecosystem Level Information
+title = "Cellana Finance"
+
+sub_ecosystems = []
+
+github_organizations = ["https://github.com/CellanaFinance/core"]
```
Comment (emrecaliskan) [inline data/ecosystems/c/data/ecosystems/m/cellana-finance.toml:6]: This is not an organization link. Could you update this to be the org link like `https://github.com/CellanaFinance`

File: data/ecosystems/a/aptos.toml [hunk 2]
```diff
@@ -17,6 +17,7 @@ sub_ecosystems = [
   "BlockPi Network",
   "Bware Labs",
   "Chainbase",
+  "Cellana Finance",
```
Comment (emrecaliskan) [inline data/ecosystems/a/aptos.toml:20]: This is failing a build check due to sort (see [here](https://github.com/electric-capital/crypto-ecosystems/actions/runs/9658311800/job/26639228291?pr=1391). Please move this above Chainbase to maintain alphabetical order.

**Review by emrecaliskan — CHANGES_REQUESTED** [review]
Thanks for the PR! Needs a couple small fixes before we can merge it.

File: data/ecosystems/c/data/ecosystems/m/cellana-finance.toml (hunk 1 above)
Comment (Centure2222) [reply data/ecosystems/c/data/ecosystems/m/cellana-finance.toml]: we updated

File: data/ecosystems/a/aptos.toml (hunk 2 above)
Comment (Centure2222) [reply data/ecosystems/a/aptos.toml:19]: we updated

**Comment by Centure2222** [comment]
please help me check

**Comment by jubos** [comment]
Thanks for the fixes!

**Pull request merged**